packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -132,7 +132,7 @@ class FormAds extends Component {
                                                         <Input
                                                             {...field}
                                                             data-lpignore='true'
-                                                            type='number'
+                                                            type='text'
```

Comment:
is it possible to make client only allowing entering number only?

Reply:
that's the exact issue we're having. allowing only number will allow input `e`.